styles/styles.css

```diff
@@ -68,35 +67,63 @@ body {
   line-height: 1.6;
   color: var(--text-color);
   background-color: var(--background-color);
-  display: none;
 }
 
-body.appear {
-  display: block;
+/* Use opacity over display: none to let the browser download LCP image assets */
+body:not([data-status='loaded']) {
```

Comment:
By using `opacity` over `display` we can assure the browser's picks up images and prioritize during load.